.github/workflows/reusable-check-labels-on-issues.yml

```diff
@@ -18,13 +18,18 @@ on:
         default: '✅ Yay, issue looks great!'
         type: string
         required: false
+      cancel-running-jobs:
+        description: 'Cancel currently in progress jobs when new ones are added.'
+        default: true
+        type: boolean
+        required: false
     secrets:
       github-token:
         required: true
 
 concurrency:
-  group: danger-${{ github.event.issue.number }}
-  cancel-in-progress: true
+  group: ${{ github.workflow }}-${{ github.event.issue.number }}
```

Comment:
Using `${{ github.workflow }}` is a neat tidying.

For my learning, I assume it's is just a tidy up, though, or was it required for the change to work as expected?